ncj/3dsmax/vray-dr/job.template.json

```diff
@@ -199,14 +216,18 @@
                         },
                         "multiInstanceSettings":{
                             "numberOfInstances":"[parameters('nodeCount')]",
-                            "coordinationCommandLine":"cmd.exe /c if \"[parameters('vrayRenderer')]\" == \"VRayAdv\" (cmd.exe /c start \"\" \"C:\\Autodesk\\3dsMax2018\\vrayspawner2018.exe\" \"-port=[parameters('vrayPort')]\") else (cmd.exe /c start C:\\server\\3dsmaxVray\\bin\\vray.exe -server -portNumber=[parameters('vrayPort')])",
+                            "coordinationCommandLine":"cmd /c %AZ_BATCH_JOB_PREP_WORKING_DIR%/PrepareVrayServices.bat --MaxVersion [parameters('maxVersion')] --VrayRenderer [parameters('vrayRenderer')] --VrayPort [parameters('vrayPort')])",
```

Comment:
Don't use these named args as you don't check for them in the batch file.  Just be clear in the batch file that the args are: <MaxVersion> <VRayRenderer> <VRayPort>